common/src/main/java/com/wynntils/services/itemvault/type/SavedItem.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemvault.type;
+
+import java.util.Set;
+
+public record SavedItem(String base64, Set<String> categories, ItemStackInfo itemStackInfo)
+        implements Comparable<SavedItem> {
+    @Override
+    public int compareTo(SavedItem other) {
+        return this.base64.compareTo(other.base64);
+    }
```

Comment:
Maybe compare the item stack info as well? Or even the categories. I don't think that would hurt.

Reply:
You can use `ComparisonChain`.

Reply:
Hm. Maybe it's not need afterall.